var/spack/repos/builtin/packages/nvshmem/package.py

```diff
@@ -38,12 +40,23 @@ class Nvshmem(MakefilePackage, CudaPackage):
         when="@2.6:",
         description="Build with support for GPU initiated communication",
     )
+    variant(
+        "ibrc",
+        default=False,
+        when="@2.6:",
+        description="Build with support for IBRC remote transport",
+    )
     conflicts("~cuda")
 
+    build_system("makefile", "cmake", default="makefile")
```

Comment:
FYI going forward from 2.9 we are making cmake the default build system for NVSHMEM (Makefile still supported in 2.9).